projects/m1/017-what-day-of-the-week/solutions/stefanoconsonni/solution.js

```diff
@@ -0,0 +1,21 @@
+let enteredYear = prompt("Please enter a year:");
+
+const dayOfTheWeek = (enteredYear + Math.floor((enteredYear - 1) / 4) - Math.floor((enteredYear - 1) / 100) + Math.floor((enteredYear - 1) / 400)) % 7;
+
+if (dayOfTheWeek == 0) {
```

Comment:
Usa sempre  === (se non ne sai il significato, guarda la documentazione a riguardo)

Inoltre, sebbene la tua soluzione funzioni, prova ad immaginarne una in cui non devi necessariamente fare ogni singolo controllo a mano.

Reply:
Ciao Simone, ti ringrazio per il feedback. Ho ascoltato i consigli e ho pushato una nuova soluzione. Che ne pensi?